juriscraper/opinions/united_states/state/pasuperct.py

```diff
@@ -15,3 +15,10 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.court_id = self.__module__
         self.url = "https://www.pacourts.us/Rss/Opinions/Superior/"
+
+    def _get_precedential_statuses(self):
+        """Mark superior court opinions Unpublished
+
+        :return: List of published statuses
+        """
+        return ["Unpublished"] * len(self.cases)
```

Comment:
Setting precedential_status as Unpublished would be incorrect, too. See some of the recent cases on the [source](https://www.pacourts.us/courts/superior-court/court-opinions) that users are probably using
![image](https://github.com/freelawproject/juriscraper/assets/14970769/d8997dbe-e7a6-4ef4-a67d-2cad4966aa02)